src/__tests__/components/ProfileMenu/ProfileMenu.test.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import '@testing-library/jest-dom/extend-expect';
+
+import Adapter from 'enzyme-adapter-react-16';
```

Comment:
Why are you using both `enzyme` and `react-testing-library`? I suggest you to choose one for the sake of consistency across the project 🍩